src/features/hotspots/settings/HotspotDiagnosticReport.tsx

```diff
@@ -213,7 +213,10 @@ const HotspotDiagnosticReport = ({ onFinished }: Props) => {
       natType: capitalize(diagnostics?.nat_type || ''),
       ip: capitalize(diagnostics?.ip || ''),
       height: info.height.toLocaleString(locale),
-      lastChallengeDate: format(fromUnixTime(info.lastChallengeTime), DF),
+      lastChallengeDate:
+        info.lastChallengeTime === 0
+          ? 'never'
```

Comment:
One thing we can do here is use translations for other languages. You only need to add the english string to the `src/locales/en.ts` file such as `never_challenged: 'never'` under the `hotspot_settings.diagnostics` object and then you can reference it using `t('hotspot_settings.diagnostics.never_challenged')`. We will then add the translations for other languages later.